### ISSUE TITLE
spin-orbital MO antisymmetrized integrals ordering

### ISSUE THREAD

**Issue by tylertak** [psi4/psi4numpy#64]
I'm looking over the psi4numpy CCSD.py code and trying to figure out the order of the integrals returned by "MO_spin = np.asarray(mints.mo_spin_eri(C, C))"

https://github.com/psi4/psi4numpy/blob/master/Coupled-Cluster/Spin_Orbitals/CCSD/CCSD.py#L79

Does Psi4 order the integrals where the alpha and beta spin functions alternate? For example, even-numbered orbitals corresponding to alpha, and odd-numbered to beta.

Also, are the integrals of the form  [phi_i(1) phi_j(1) 1/r_12 phi_k(2) phi_l(2)] or <phi_i(1) phi_k(2) 1/r_12 phi_j(1) phi_l(2)>?

Thank you

**Comment by vijaymocherla** [comment]
> Does Psi4 order the integrals where the alpha and beta spin functions alternate? For example, even-numbered orbitals corresponding to alpha, and odd-numbered to beta.

Yes, it starts with alpha electrons, and goes on to beta. even/odd indexing of alpha/beta happens because the indexing starts from 0 for arrays in python (that's also the case for C/C++, but not Fortran) 

> Also, are the integrals of the form [phi_i(1) phi_j(1) 1/r_12 phi_k(2) phi_l(2)] or <phi_i(1) phi_k(2) 1/r_12 phi_j(1) phi_l(2)>?

If this is a question regarding the convention being used, then the integrals obtained are in chemist's notation i.e. 
$$[ij|kl] = \int dx_{1} \int dx_{2} \   \chi_{i}^{\*}(x_1) \chi_j(x_1) \frac{1}{r_{12}} \chi_{k}^{\*} (x_2) \chi_l(x_2)$$

where coordinate $x$  is used to represent both spatial coordinates $r$ and spin $\omega$ of the electron. (For reference see Szabo and Ostlund)
